frontend/app/pods/admin/teams/template.hbs

```diff
@@ -7,6 +7,7 @@
 <div class="row">
   <div class="small-5 columns">
     {{#each model as |team|}}
+      {{#if team.name}}
```

Comment:
So there might be another problem. It shouln't be the way you describe it. 
This sort of hides that bug so I prefer to take it out and understand what is the underlying problem